t/e2e/rpc/rpc_test.go

```diff
@@ -144,6 +144,9 @@ func (s *RPCTestSuite) TestCallRawResult() {
 // for a not yet mained transaction is "error":{"code":-32000,"message":"unknown transaction"}.
 // Issue: https://github.com/status-im/status-go/issues/547
 func (s *RPCTestSuite) TestCallRawResultGetTransactionReceipt() {
+	if SkipTransactionTest() {
```

Comment:
Just checking the proper error output. It should be fine on both StatusChain and mainnet.